### PR TITLE
refactor(ontology)!: use patch instead of delete for deleting cardinalities (DSP-1700)

### PR DESCRIPTION
**Pull request by irinaschubert**
resolves DSP-1700

### CODE REVIEW

**Comment by irinaschubert** [comment]
Hi @subotic , I used PATCH instead of DELETE for the deletion of cardinalities (PUT was already in use). It seems that PATCh wasn't used before so I had to enable it and also adjust the tests for these methods. I assigned the PR to you just that you can see what I did. You don't need to review it as you are on holidays.

**Review by kilchenmann — APPROVED** [review]
LGTM. Thank you

File: webapi/src/test/scala/org/knora/webapi/e2e/CORSSupportE2ESpec.scala [hunk 1]
```diff
@@ -51,30 +51,31 @@ class CORSSupportE2ESpec extends E2ESpec(CORSSupportE2ESpec.config) {
...
-      responseToString(response) shouldEqual "CORS: invalid method 'PATCH'"
-      response.status shouldBe StatusCodes.BadRequest
-    }
+//    "reject requests with invalid method" in {
+//      val request = Options(baseApiUrl + s"/admin/projects") ~> Origin(exampleOrigin) ~> `Access-Control-Request-Method`(
+//        PATCH)
+//      val response: HttpResponse = singleAwaitingRequest(request)
+//      responseToString(response) shouldEqual "CORS: invalid method 'PATCH'"
+//      response.status shouldBe StatusCodes.BadRequest
+//    }
```
Comment (BalduinLandolt) [inline webapi/src/test/scala/org/knora/webapi/e2e/CORSSupportE2ESpec.scala:72]: can this be removed?

**Review by BalduinLandolt — APPROVED** [review]
looks good to me

File: webapi/src/test/scala/org/knora/webapi/e2e/CORSSupportE2ESpec.scala (hunk 1 above)
Comment (irinaschubert) [reply webapi/src/test/scala/org/knora/webapi/e2e/CORSSupportE2ESpec.scala]: You are right. It's probably confusing in the future to have uncommented code blocks. I removed it.

**Pull request merged**